packages/web/components/bridge/immersive/amount-and-review-screen.tsx

```diff
@@ -103,8 +105,12 @@ export const AmountAndReviewScreen = observer(
           ? fromAsset?.supportedVariants[toAsset?.address ?? ""]
           : toAsset?.supportedVariants[fromAsset?.address ?? ""],
       onTransfer: () => {
+        setToAsset(undefined);
+        setFromAsset(undefined);
         setCryptoAmount("0");
         setFiatAmount("0");
+
+        refetchUserQueries(apiUtils);
```

Comment:
Makes sure balances are accurate when restarting the d/w flow

Reply:
Nit: I believe we should add this same GH comment as a code comment for this line